### ISSUE TITLE
Unsupported Athena type: array<bigint>

### ISSUE THREAD

**Issue by robkano** [awslabs/aws-data-wrangler#206]
**Describe the bug**
Using a datafame file that has columns with arrays of numbers (int64) or string. 
Trying to append new data to the existing table.
This is happened with mode=`append` or `overwrite_partitions`. 
**To Reproduce**
```
df = pd.DataFrame({
        "id": [1, 2, 3, 4],
        "value": [["foo", "boo", "bla"], ["dkajf", "lhjasdfue", "kja", "jdf"], ["kkk"], []],
        "collaborator_id": [[666665554477, 8888888873, 55584839993, 22333300000], [74649993333, 4774332222], [77466392002, 663666662222], [8883999999]]
    })

wr.s3.to_parquet(
    df=df,
    path=s3://path ...,
    dataset=True,
    mode="append",
    database='your_db',
    table='table_name'
    boto3_session=boto3_session
)
```

**Comment by igorborgest** [comment]
@robkano thanks a lot for report it.

I've just added support for nested arrays and structs. Will be released soon in the next version (`1.1.0`).

**Comment by robkano** [comment]
@igorborgest Thanks. Good job you did

**Comment by igorborgest** [comment]
Available on version [1.1.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.1.0)
